cg_hermes/config/tomte.py

```diff
@@ -0,0 +1,137 @@
+"""Tags that are defined in Tomte deliverables mapped to tags used in CG
+
+The tag sets that exists in all files are set to mandatory. Tag sets that exists in < 4 deliverables are not
+mandatory by default. However, the tags that are available to a particular analysis is mandatory for that analysis.
+"""
+
+from cg_hermes.config.nextflow import NEXTFLOW_TAGS
+
+TOMTE_COMMON_TAGS = {
+    frozenset({"fraser", "fraser-clinical"}): {
+        "is_mandatory": False,
+        "tags": ["fraser", "clinical"],
+        "used_by": ["deliver"],
+    },
+    frozenset({"fraser", "fraser-research"}): {
+        "is_mandatory": False,
+        "tags": ["fraser", "research"],
+        "used_by": ["deliver"],
```

Comment:
Have we identified which ones we're keeping for long term storage?

Reply:
not yet, I will ask and add `long-term-storage` for those

Reply:
Sweet 🍬

Reply:
added!